ncm-spma 14.10.0-rc1: missing dependency for Text/Glob-pm

Blocking issue for 14.10.0 (https://github.com/quattor/release/issues/58).

```
2014/10/30-13:22:00 -----------------------------------------------------------
2014/10/30-13:22:00 [ERROR] Failed to load NCM::Component::spma::yum: Can't locate Text/Glob.pm in @INC (@INC contains: /usr/lib/perl /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5) at /usr/lib/perl/NCM/Component/spma/yum.pm line 34.
BEGIN failed--compilation aborted at /usr/lib/perl/NCM/Component/spma/yum.pm line 34.
Compilation failed in require at (eval 27) line 2.
BEGIN failed--compilation aborted at (eval 27) line 2.
```

--- a/ncm-spma/pom.xml
+++ b/ncm-spma/pom.xml
@@ -209,6 +209,7 @@
               <require>perl(LC::Exception)</require>
               <require>perl(NCM::Component)</require>
               <require>perl(Set::Scalar)</require>
+              <require>perl(Text::Glob)</require>
               <require>perl(constant)</require>
               <require>perl(strict)</require>
               <require>perl(warnings)</require>

